templates/static-files.conf

```diff
@@ -0,0 +1,60 @@
+server {
+        listen 80;
+        server_name {{ site_url }};
+
+        {% if use_ssl %}
+        return 301 https://{{ site_url }}$request_uri;
+        {% else %}
+        root {{ site_path }};
+
+        location / {
+                index index.html;
+        }
+
+        include /etc/nginx/locations.d/{{ site_url }}/*.conf;
+        {% endif %}
+}
+
+{% if use_ssl %}
+server {
+        listen 443 ssl http2;
+        listen [::]:443 ssl http2;
+        server_name {{ site_url }};
+
+        # certs sent to the client in SERVER HELLO are concatenated in ssl_certificate
+        ssl_certificate {{ ssl_certificate_path }};
+        ssl_certificate_key {{ ssl_key_path }};
+        ssl_session_timeout 1d;
```

Comment:
This should become a variable that is set with a default value in defaults/meta.yml